### PR TITLE
fixing minor grammar and typos

### PR DESCRIPTION
**Pull request by Zolgarr**

### CODE REVIEW

File: docs/why-wasabi/LegalConcern.md [hunk 1]
```diff
@@ -11,8 +11,8 @@
 
 ---
 
-Using Bitcoin and other cryptocurrencies in the everyday course of businesses is becoming more and more popular.
-Digital money has a lot of advantages, from unstoppable transactions, to low fees, and instant execution of international payments.
+Using Bitcoin and other cryptocurrencies in the everyday course of businesses are becoming more and more popular.
```
Comment (yahiheb) [inline docs/why-wasabi/LegalConcern.md:14]: I think `is` is correct here but I am not sure.
How about something like this:
```suggestion
Bitcoin and other cryptocurrencies usage in the everyday course of businesses is becoming more and more popular.
```

Comment (MaxHillebrand) [reply docs/why-wasabi/LegalConcern.md:14]: It is `is`, not `are`.

File: docs/why-wasabi/LegalConcern.md [hunk 2]
```diff
@@ -71,12 +71,12 @@ Other competitors can now demand similar discounts, forcing industries into a vi
 As each and every price discount is negotiated on a case by case basis, considering factors as creditworthiness and timing of payment, transparent supplier prices would only cause enormous pressure on suppliers to deal with negotiations.
 
 Beyond the price of products, the timing of payments is also fully transparent on the blockchain.
-However, in certain circumstances, it is also a crucial business secret to keep the information of the sequence of invoice payments indoors.
+However, in certain circumstances, it is also a crucial business secret to keep the information on the sequence of invoice payments indoors.
```
Comment (MaxHillebrand) [inline docs/why-wasabi/LegalConcern.md:74]: I think `of` is correct here?

File: docs/why-wasabi/LegalConcern.md (hunk 1 above)
Comment (Zolgarr) [reply docs/why-wasabi/LegalConcern.md:14]: Indeed, you're right, my bad!

File: docs/why-wasabi/LegalConcern.md (hunk 2 above)
Comment (Zolgarr) [reply docs/why-wasabi/LegalConcern.md:74]: I think `of` works too, but I think it's more appropriate to use `on` or `about`
When using `about`, then `the` should be left out:
"To keep information about the sequence of invoice payments indoors."

**Pull request merged**